tests/unit/extract/test_group_holdings.py

```diff
@@ -0,0 +1,324 @@
+                            cost=CurrencyAndAmount(
+                                amount=10000,
+                                currency="AUD"
+                            ),
+                            cost_portfolio_ccy=CurrencyAndAmount(
+                                amount=0,
+                                currency="AUD"
+                            )
+                        )
+                    ]
+                ),
+                "MyScope : PortfolioB": MockVersionedResourceListOfPortfolioHolding(
+                    portfolio_holdings=[
+                        PortfolioHolding(
+                            instrument_uid="LUID_12345678",
+                            properties={
+                                "MyProperty": ModelProperty(
+                                    key="MyProperty",
+                                    value=PropertyValue(
+                                        label_value="blah"
+                                    )
+                                )
+                            },
+                            holding_type="P",
+                            units=150,
+                            settled_units=150,
+                            cost=CurrencyAndAmount(
+                                amount=10000,
+                                currency="AUD"
+                            ),
+                            cost_portfolio_ccy=CurrencyAndAmount(
+                                amount=0,
+                                currency="AUD"
+                            )
+                        )
+                    ]
+                )
+            },
+            False,
+            "PortGroupScope : MyPortGroup",
+            {
+                "PortGroupScope : MyPortGroup": MockVersionedResourceListOfPortfolioHolding(
+                    portfolio_holdings=[
+                        PortfolioHolding(
+                            instrument_uid="LUID_12345678",
+                            properties={
+                                "Instrument/default/Name": ModelProperty(
+                                    key="Instrument/default/Name",
+                                    value=PropertyValue(
+                                        label_value="Apple"
+                                    )
+                                )
+                            },
+                            holding_type="P",
+                            units=250,
+                            settled_units=250,
+                            cost=CurrencyAndAmount(
+                                amount=20000,
+                                currency="AUD"
+                            ),
+                            cost_portfolio_ccy=CurrencyAndAmount(
+                                amount=0,
+                                currency="AUD"
+                            )
+                        )
+                    ]
+                )
+            }
+        ]
+    ])
```

Comment:
I think we need a test for holdings in different currencies

Reply:
Done.